storage/aptosdb/src/backup/restore_utils.rs

```diff
@@ -183,13 +187,18 @@ pub fn save_transactions_impl(
     txns: &[Transaction],
     txn_infos: &[TransactionInfo],
     events: &[Vec<ContractEvent>],
+    write_sets: &[WriteSet],
     batch: &mut SchemaBatch,
 ) -> Result<()> {
     for (idx, txn) in txns.iter().enumerate() {
         transaction_store.put_transaction(first_version + idx as Version, txn, batch)?;
     }
     ledger_store.put_transaction_infos(first_version, txn_infos, batch)?;
     event_store.put_events_multiple_versions(first_version, events, batch)?;
+    // save the writeset to the transaction_store
```

Comment:
nit: the comment is not accurate. It's actually using a function in the transaction store to insert those changes into the batch.